ml-agents-envs/mlagents_envs/environment.py

```diff
@@ -147,55 +147,64 @@ def __init__(
     def get_communicator(worker_id, base_port, timeout_wait):
         return RpcCommunicator(worker_id, base_port, timeout_wait)
 
-    def executable_launcher(self, file_name, docker_training, no_graphics, args):
+    @staticmethod
+    def environment_launch_check(env_path):
```

Comment:
minor / subjective: maybe this method could be called `validate_environment_path`

Reply:
Changed